docs/vercel.json

```diff
@@ -1,11 +1,11 @@
 {
   "rewrites": [
     {
-      "source": "/docs-v2",
```

Comment:
Just since we're linking to it in a few places already in the wild, can you keep a rule for `/docs-v2` and do a permanent redirect to `/docs/v2`? https://vercel.com/support/articles/does-vercel-support-permanent-redirects

Then we'll want to keep the `pipedream-proxy` entry for `/docs-v2` (in addition to adding `/docs/v2`).

Since it's a permanent redirect, I have a feeling we can remove that code after some time and Google will know how to handle `/docs-v2` (I am not sure how that works).

Reply:
This comment is technically a duplicate of the one I put in the `pipedreamin` repo. As long as we handle `/docs-v2` in one place, that is sufficient.